package.json

```diff
@@ -16,6 +16,7 @@
     "@react-three/drei": "^9.96.5",
     "@react-three/fiber": "^8.15.15",
     "@sentry/nextjs": "^7.98.0",
+    "@tabler/icons-react": "^3.0.1",
```

Comment:
No need for this! We are using font awesome, we have access to the pro icons

Reply:
done